Formula/khal.rb

```diff
@@ -64,46 +74,74 @@ class Khal < Formula
   end
 
   resource "vdirsyncer" do
```

Comment:
No; good thought, but the Homebrew formula provides the app but does not expose the library.